lib/clusterable/ht_item.rb

```diff
@@ -4,56 +4,52 @@
 require "services"
 
 module Clusterable
-  # An HT Item
-  # - ocns
-  # - item_id
-  # - ht_bib_key
-  # - rights
-  # - bib_fmt
-  # - enum_chron
-  # - n_enum
-  # - n_chron
-  # - collection_code
-  # - billing_entity
   class HtItem
     include EnumChron
-    # field :ocns, type: Array, default: []
-    # field :item_id, type: String
-    # field :ht_bib_key, type: Integer
-    # field :rights, type: String
-    # field :access, type: String
-    # field :bib_fmt, type: String
-    # field :enum_chron, type: String, default: ""
-    # field :n_enum, type: String, default: ""
-    # field :n_chron, type: String, default: ""
-    # field :n_enum_chron, type: String, default: ""
-    # field :collection_code, type: String
-    # field :billing_entity, type: String
-
-    # embedded_in :cluster
-    # validates :item_id, uniqueness: true
-    # validates_presence_of :item_id, :ht_bib_key, :rights, :bib_fmt, :access
-
-    # validates_each :ocns do |record, attr, value|
-    #   value.each do |ocn|
-    #     record.errors.add attr, "must be an integer" \
-    #       unless (ocn.to_i if /\A[+-]?\d+\Z/.match?(ocn.to_s))
-    #   end
-    # end
-
-    def initialize(params = nil)
-      raise "not implemented"
-      super
-      set_billing_entity if collection_code
+
+    def self.accessor_attrs
```

Comment:
I'd rather rename this `attr_accessors`, and have a similar pattern for `attr_readers`, for consistency.

```ruby
def self.attr_accessors
  [:ocns, :item_id, :ht_bib_key, :rights, :access, :bib_fmt, :n_enum, :n_chron, :n_enum_chron, :billing_entity, :enum_chron]
end

def self.attr_readers
  [:collection_code]
end

def self.all_attrs
  attr_accessors + attr_readers
end

attr_accessors.each { |attr| attr_accessor attr }
attr_readers.each { |attr| attr_reader attr }
```

Reply:
Done (and changed to use constants)